src/RenderingSystem.cpp

```diff
@@ -17,19 +19,23 @@ void RenderingSystem::update() {
 void RenderingSystem::load(const Event& e) {
     init_window();
 
-    example_shader_.load_shader(
-        "assets/shaders/SimpleVertexShader.vertexshader",
-        "assets/shaders/SimpleFragmentShader.fragmentshader");
-
-    MeshAsset* mesh = asset_manager_.get_mesh_asset("assets/models/Ship.obj");
+    MeshAsset* mesh = asset_manager_.get_mesh_asset("assets/models/teapot.obj");
+    TextureAsset* texture = asset_manager_.get_texture_asset("assets/textures/default.png");
+    ShaderAsset* shader = asset_manager_.get_shader_asset("assets/shaders/SimpleShader");
 
     example_objects_.emplace_back();
     example_objects_[0].set_mesh(mesh);
+    example_objects_[0].set_texture(texture);
+    example_objects_[0].set_shader(shader);
```

Comment:
I think it would be better to combine the calls.

```c++
example_objects_[0].set_shader(asset_manager_.get_shader_asset("assets/shaders/SimpleShader"));
```

No error checking is happening here anyways :P